elide-core/src/main/java/com/yahoo/elide/core/filter/Predicate.java

```diff
@@ -43,6 +44,14 @@ public Predicate(PathElement pathElement, Operator op, List<Object> values) {
         this(Collections.singletonList(pathElement), op, values);
     }
 
+    public Predicate(PathElement pathElement, Operator op){
+        this(Collections.singletonList(pathElement), op, Collections.emptyList());
+    }
+
+    public Predicate(List<PathElement> path, Operator op){
+        this(path, op, Collections.emptyList());
+    }
+
```

Comment:
These were recently removed by @xiaoyao1991's PR. Are they necessary to your PR or do you need to rebase?